.travis.yml

```diff
@@ -8,7 +8,11 @@ dist: xenial
 jdk:
   - openjdk11
   - openjdk15
-install: true
+addons:
+  firefox: latest
+install:  
+  - if [ "$TRAVIS_OS_NAME" == "linux" ]; then wget -N https://github.com/mozilla/geckodriver/releases/download/v0.26.0/geckodriver-v0.26.0-linux64.tar.gz -P ~/ ; tar -xzf ~/geckodriver-v0.26.0-linux64.tar.gz -C ~/ ; rm ~/geckodriver-v0.26.0-linux64.tar.gz ; sudo mv -f ~/geckodriver /usr/local/share ; sudo chmod +x /usr/local/share/geckodriver ; sudo ln -s /usr/local/share/geckodriver /usr/local/bin/geckodriver ; fi
```

Comment:
Can we have a URL which is not dependent upon a specific version? otherwise we'll need to manage this package version as well

Reply:
I will look into this

Reply:
I just found [webdriver manager](https://github.com/bonigarcia/webdrivermanager) which deals with downloading the driver. Also looks actively maintained